x-pack/plugins/security_solution/public/management/common/utils.ts

```diff
@@ -19,3 +21,31 @@ export const parseQueryFilterToKQL = (filter: string, fields: Readonly<string[]>
 
   return `(${kuery})`;
 };
+
+const getPolicyQuery = (policy: string): string => {
+  if (policy === 'global') return 'exception-list-agnostic.attributes.tags:"policy:all"';
+  if (policy === 'unassigned') return '(not exception-list-agnostic.attributes.tags:*)';
+  return `exception-list-agnostic.attributes.tags:"policy:${policy}"`;
+};
+
+export const parsePoliciesToKQL = (includedPolicies: string, excludedPolicies: string): string => {
+  if (isEmpty(includedPolicies) && isEmpty(excludedPolicies)) return '';
+
+  const parsedIncludedPolicies = includedPolicies ? includedPolicies.split(',') : undefined;
+  const parsedExcludedPolicies = excludedPolicies ? excludedPolicies.split(',') : undefined;
```

Comment:
I did not yet get to the subsquent code, but wondering why we need `excludedPolicies`. By default, all is included and if `includedPolicies` were defined, then only those should be returned, thus the rest is already excluded. Right?

Reply:
Hey! In order to keep url clean, I'm adding `includePolicies` when you want to filter by those and `excludePolicies` when you don't want to filter by those. So, when `excludePolicies` and `includePolicies` are empty, all the existing results are returned. 
When you want to get only TA with policy 1 and 2, those policies are added to the `includePolicies`. 
If you want to exclude those who have policy 3, then this is added to the `excludePolicies`. 

As a TA can have zero, one or more policies assigned I thought this was a good way to do this. Do you have any other?

Reply:
I just realized that each menu item is enabled to have 3 states: selected, unselected and excluded. Sorry - I did not realize that before.

I do wonder if that is the desired UX for this menu item - to support an "excluded" list _(FYI: the fleet policy selector filter on the agents list only allows selected and not selected)_. You can follow up on that with @bfishel and/or @caitlinbetz on that and address later if needed

Reply:
Thanks @dasansol92 . You did not have to do that - could have just merged and then be adjusted later if needed. but thanks again 😸